apps/survey/contracts/Survey.sol

```diff
@@ -45,8 +46,8 @@ contract Survey is AragonApp {
         uint256 participation;                  // tokens that casted a vote
         string metadata;
 
-        mapping (uint256 => uint256) votes;     // option -> voting power for option
-        mapping (address => MultiOptionVote) voters;    // voter -> options voted, with its stakes
+        mapping (uint256 => uint256) optionVotes;     // option -> voting power for option
```

Comment:
You're probably going to kill me 🙈, but what do you think about just renaming this `optionPower`? `optionVotes` gets me too confused with `votes` half the time.

Reply:
No problem, next time just don't give me options to choose! Hahaha.